ui/src/app/pages/workspace/workspace-list.tsx

```diff
@@ -117,7 +117,11 @@ export const WorkspaceList = fp.flow(withUserProfile())(
                   options={filters}
                   defaultValue={defaultFilter}
                   onChange={({ value }) =>
```

Comment:
This onChange seems to be returning the selected value rather than the updated array. That means that if you select 'All', the value would be null and map would not work.